wagtailio/project_styleguide/templates/patterns/pages/showcase_page/showcase_page.html

```diff
@@ -0,0 +1,53 @@
+{% extends "patterns/base_page.html" %}
+
+{% load static i18n wagtailcore_tags wagtailembeds_tags %}
+
+{% block extra_js %}
+    <script src="{% static 'js/htmx.js' %}" defer></script>
+{% endblock extra_js %}
+
+{% block body_class %}template-showcase{% endblock %}
+
+{% block title %}Wagtail CMS - {% firstof page.seo_title page.title %}{% endblock %}
+{% block title_suffix %}{% endblock %}
+
+{% block content %}
+    <div class="showcase-page__hero showcase-page__content">
+        <h1 class="showcase-page__title">{{ page.title }}</h1>
+        {% if page.introduction %}<div class="showcase-page__intro">{{ page.introduction }}</div>{% endif %}
+    </div>
+    <div class="showcase-page__content">
+        <form
```

Comment:
question: what happens if people have no JS?

Reply:
Fair question, as of late, I'm less and less concerned with non-JS, especially due to being deproritised by GDS. Although, the only real annoyance this requires is a small amount of pop-out from hidding a submit button, which we can do.